Makefile

```diff
@@ -0,0 +1,27 @@
+# Copyright 2022 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+precommit: FORCE tidy lint csv
+
+csv: FORCE
+	go run . csv github.com/google/go-licenses
+
+lint: FORCE
+# There's no need to lint third_party packages, so we explicitly list other packages here.
+	golint --set_exit_status . licenses/...
```

Comment:
Few things here:

1. Looks like [golint is deprecated](https://github.com/golang/lint/blob/6edffad5e6160f5949cdefc81710b2706fbcd4f6/README.md?plain=1#L1-L3)
2. We're using [golangci-lint](https://github.com/golangci/golangci-lint) for our [presubmits](https://github.com/google/go-licenses/blob/master/.github/workflows/golangci-lint.yml) (idk if there would be any skew)

Reply:
Switched to golangci-lint instead.